src/main/drivers/max7456.c

```diff
@@ -256,6 +260,11 @@ static void max7456SendDma(void* tx_buffer, void* rx_buffer, uint16_t buffer_siz
     DMA_Init(MAX7456_DMA_CHANNEL_TX, &DMA_InitStructure);
     DMA_Cmd(MAX7456_DMA_CHANNEL_TX, ENABLE);
 
+#if defined(STM32F4) && !defined(MAX_DMA_CHANNEL_RX)
+    // XXX DMA doesn't start without this when using TX channel only!?
+    MAX7456_DMA_CHANNEL_TX->CR;
```

Comment:
This is the magic spell that made DMA work without RX channel. Without this line, DMA will not start. Theory behind this behavior is wanted.